.github/workflows/build.yml

```diff
@@ -0,0 +1,36 @@
+name: build
```

Comment:
Add comment:

# Copyright (c) Microsoft Corporation. All rights reserved.
# Licensed under the MIT License.

Reply:
I don't think this config file is in any way copyrightable, but I'll add it.